test/e2e/quadlet/basic.network

```diff
@@ -0,0 +1,7 @@
+## assert-key-is Unit RequiresMountsFor "%t/containers"
+## assert-key-is Service Type oneshot
+## assert-key-is Service RemainAfterExit yes
+## assert-key-is Service ExecStart "/usr/bin/podman network create systemd-basic"
```

Comment:
This will conflict with https://github.com/containers/podman/pull/16688 (needs a regexp check to handle podman outside /usr/bin/podman.

Reply:
Sorry, will conflict with #16825 (due to the /usr/bin/podman vs $BINDIR/podman)

Reply:
Fixed following the merge of #16862